resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -25,7 +25,7 @@
   </parent>
 
   <artifactId>spark-kubernetes-integration-tests_2.11</artifactId>
-  <groupId>spark-kubernetes-integration-tests</groupId>
```

Comment:
I am wondering if we need `groupId `?

Reply:
> I am wondering if we need groupId?

Yes. Each project must have a `groupId` and `artifactId`.

Reply:
Discussed offline. `groupId` can be ignored and it will inherit from the parent module. I just removed it.